Playbooks/playbook-URL_Enrichment_-_Generic.yml

```diff
@@ -0,0 +1,235 @@
+      }
+  "23":
+    id: "23"
+    taskid: 173d4bc7-468c-4751-8137-50e8adb50220
+    type: title
+    task:
+      id: 173d4bc7-468c-4751-8137-50e8adb50220
+      version: -1
+      name: URL Indicators
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "16"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 195
+        }
+      }
+  "24":
+    id: "24"
+    taskid: b3f42dba-eebb-4744-8929-879c0427bbf7
+    type: title
+    task:
+      id: b3f42dba-eebb-4744-8929-879c0427bbf7
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 863
+        }
+      }
+  "25":
+    id: "25"
+    taskid: 7805ced2-6142-4f98-8a34-d36b3df4a36c
+    type: condition
+    task:
+      id: 7805ced2-6142-4f98-8a34-d36b3df4a36c
+      version: -1
+      name: 'Is Rasterize integration enabled? '
+      description: Check if we can use rasterize
+      scriptName: Exists
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "24"
+      "yes":
+      - "26"
+    scriptarguments:
+      value:
+        complex:
+          root: modules
+          filters:
+          - - operator: string.isEqual
+              left:
+                value:
+                  simple: brand
+                iscontext: true
+              right:
+                value:
+                  simple: Rasterize
+          - - operator: string.isEqual
+              left:
+                value:
+                  simple: state
+                iscontext: true
+              right:
+                value:
+                  simple: active
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 362.5,
+          "y": 511
+        }
+      }
+  "26":
+    id: "26"
+    taskid: eabb6065-b788-45b2-8a74-99f2598a7484
+    type: regular
+    task:
+      id: eabb6065-b788-45b2-8a74-99f2598a7484
+      version: -1
+      name: Get URL screenshot
+      description: Get the screenshot of the url page
+      script: Rasterize|||rasterize
+      type: regular
+      iscommand: true
+      brand: Rasterize
+    nexttasks:
+      '#none#':
+      - "24"
+    scriptarguments:
+      height: {}
+      type: {}
+      url:
+        simple: ${URL.Data}
+      width: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 488,
+          "y": 698
+        }
+      }
+view: |-
+  {
+    "linkLabelsPosition": {
+      "16_20_yes": 0.79,
+      "16_24_no": 0.15
+    },
+    "paper": {
+      "dimensions": {
+        "height": 878,
+        "width": 1145,
+        "x": -277,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: url
+  value: {}
+  required: true
+  description: url to enrich
+outputs:
+- contextPath: URL.Address
```

Comment:
Add the `URL` output as well (with out it, if any integration add other attributes like `Malicious` it won't return to the main playbook)

Reply:
added malicious and its sub-labels (vendor, description)

Reply:
please add `URL` as well. o/w the data won't return from relevant integrations